### PR TITLE
Change manifest person pages to use same page as saved people

### PR DESCRIPTION
**Pull request by Meepit**
Use given name & surname consistently over first name & last name

Content changes

**What changes does this PR bring?**

**Definition of Done**
The following need to be checked off as they are complete as part of the
PR process. If you are unable to mark off an item as complete, then state
in the comments why this is the case.
- [ ] Have you built the code locally and confirmed its working?
- [ ] Is the build confirmed to be passing on CI?
- [ ] Have you added enough relevant unit tests for your change?
- [ ] Have you added enough relevant E2E tests for your change?
- [ ] Have you updated any relevant documentation as part of this change?
- [ ] Has this change been tested against the Acceptance Criteria?
- [ ] Have you considered how this will be deployed in SIT/Staging/Production?

### CODE REVIEW

File: src/app/garfile/manifest/editperson/index.njk [hunk 1]
```diff
@@ -30,211 +30,22 @@
...
-                  <input class="govuk-input govuk-date-input__input govuk-input--width-4" id="dobyear" name="dobyear" type="number" value="{{cookie.dateSlice('year', personDetails.dateOfBirth)}}" pattern="[0-9]*">
-                </div>
-              </div>
-        </fieldset>
-      </div>
-
-    <div class="govuk-form-group {{ m.form_group_class('persontype', errors) }}">
-      <label class="govuk-label govuk-label--m" for="persontype">Person Type</label>
-      <span class="govuk-hint">
-        For example, Captain, Crew or Passenger. First
```
Comment (SvetlaVerma) [inline src/app/garfile/manifest/editperson/index.njk]: Why are we deleting this James? One of the suggestion in "Add person details" is around this hint text:
"For example, Captain, Crew or Passenger. First Office / Co-Pilot should be identified as Crew."

File: src/app/garfile/manifest/editperson/index.njk [hunk 2]
```diff
@@ -30,211 +30,22 @@
...
-        Office / Co-Pilot should be identified as Crew.
-      </span>
-      {{ m.error_message('persontype', errors) }}
-      <select class="govuk-select" id="persontype" name="persontype">
-        {% for person in travepersontype %}
```
Comment (SvetlaVerma) [inline src/app/garfile/manifest/editperson/index.njk]: Travel Document Details field is not visibile in the "Add person details" page anymore.

File: src/app/garfile/supportingdocuments/get.controller.js [hunk 3]
```diff
@@ -19,7 +19,7 @@ module.exports = (req, res) => {
           res.render('app/garfile/supportingdocuments/index', { cookie, supportingDoc, errors: [{ message: 'File cannot be uploaded. The file has a virus.'}]})
         } else if (req.query.query === '0') {
           logger.debug('no file')
```
Comment (SvetlaVerma) [inline src/app/garfile/supportingdocuments/get.controller.js]: Pressing "Save and exit" on supporting documents page, gives an error. The screen shot of that I have shared.

Comment (SvetlaVerma) [reply src/app/garfile/supportingdocuments/get.controller.js]: Also get the same error when i select a GAR in a list of Draft GARs.

File: src/common/templates/includes/person-details.njk [hunk 4]
```diff
@@ -158,6 +161,9 @@
 <div class="govuk-form-group {{ m.form_group_class('issuing-state', errors) }}">
   <label class="govuk-label govuk-label--m" for="issuing-state">Issuing state</label>
   {{ m.error_message('issuing-state', errors) }}
+  <span class="govuk-hint">
+    Provide the 3 letter code for the issuing state, for example GBR
```
[the remark this answers is not in the document]
Comment (Meepit) [reply src/common/templates/includes/person-details.njk]: Comments for page 3 of the service suggested we use:
-	Add prompt text to ensure people know to use 3 letter codes: ‘Provide the 3 letter code for the issuing state, for example GBR’.

File: src/app/garfile/manifest/editperson/index.njk (hunk 1 above)
Comment (Meepit) [reply src/app/garfile/manifest/editperson/index.njk]: I've readded this

File: src/app/garfile/manifest/editperson/index.njk (hunk 2 above)
Comment (Meepit) [reply src/app/garfile/manifest/editperson/index.njk]: The page has been changed to be the same as the saved person page, this was always the intention. The current manifest page's layout does not facilitate the changing of content as suggested

File: src/app/aircraft/add/index.njk [hunk 5]
```diff
@@ -76,10 +76,10 @@
       </div>
```
[the remark this answers is not in the document]
Comment (Meepit) [reply src/app/aircraft/add/index.njk]: Fixed

File: src/app/garfile/supportingdocuments/get.controller.js (hunk 3 above)
Comment (Meepit) [reply src/app/garfile/supportingdocuments/get.controller.js]: Fixed

**Pull request merged**